packages/botonic-plugin-contentful/src/contentful/manage/manage-contentful.ts

```diff
@@ -60,17 +60,27 @@ export class ManageContentful implements ManageCms {
     context: ManageContext,
     contentId: ContentId,
     fieldType: ContentFieldType,
-    value: any
+    value: any,
+    code?: string
   ): Promise<void> {
     const environment = await this.getEnvironment()
     const oldEntry = await environment.getEntry(contentId.id)
+
     const field = this.checkOverwrite(context, oldEntry, fieldType, true)
     oldEntry.fields[field.cmsName][context.locale] = value
+    if (code) this.replaceCode(code, oldEntry)
+
     // we could use this.deliver.contentFromEntry & IgnoreFallbackDecorator to convert
     // the multilocale fields returned by update()
     await this.writeEntry(context, oldEntry)
   }
 
+  replaceCode(code: string, entry: Entry) {
+    for (const locale in entry.fields.name) {
```

Comment:
since code is not localizable, I'm surprised this works

Reply:
Yes, it's not localizable, but this is how `entry.fields.name` is formed, is an object with a `key: value` pair where the `key` is the default locale.
I did the `for` just in case the Code field was localizable.